src/components/ratingStatsCard.js

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from "react";
+import { getUserRatingStats } from "../backend";
+
+export function RatingStatsCard(props) {
+  const [stats, setStats] = useState(null);
+
+  useEffect(() => {
+    (async () => {
+      setStats(await getUserRatingStats(props.userId));
+    })();
+  }, []);
+
+  return (
+    <>
+      { stats !== null &&
+        <>
+        <p className="my-2 fg-brand-blue">{stats.totalRatings} total reviews</p>
+        <p>Average Rating: </p>
+        <div class="stars" style={{"--rating": `${Number(stats.averageAll) / 5.0}`}}>
+          <div class="bg">⭐⭐⭐⭐⭐</div>
+        </div>
+        <p>Communication: </p>
+        <div class="stars" style={{"--rating": `${Number(stats.averageCommunication) / 5.0}`}}>
+          <div class="bg">⭐⭐⭐⭐⭐</div>
+        </div>
+        <p>Exchange: </p>
+        <div class="stars" style={{"--rating": `${Number(stats.averageExchange) / 5.0}`}}>
+          <div class="bg">⭐⭐⭐⭐⭐</div>
+        </div>

```

Comment:
How does this work to make partial stars? I'm just curious. I might rework the reviews card to have stars like this

Reply:
In the CSS there is this code:
```
.stars {
  position: relative;
  display: inline-block;
}

.bg {
  opacity: 0.35;
  filter: grayscale(1);
}

.stars::after {
  content: '⭐⭐⭐⭐⭐';
  position: absolute;
  top: 0;
  left: 0;
  width: calc(100% * var(--rating));
  overflow: hidden;
}
```

Essentially the bg stars will be made to look grey and the .stars:after will overlay the coloured stars on top with their width of the shown stars to be proportional to the rating percentage. It seems to work well. The effect can be seen in the screenshot